src/services/defaultClasses.js

```diff
@@ -16,6 +16,7 @@ export default {
     },
     selected: {
         'vue-ads-bg-teal-100': true,
+        'vue-ads-bg-gray-100': false,
```

Comment:
Is this necessary?

Reply:
Yes, that's why you can not select even records, actually it selected but user eye don't see it because of styling, we need to disable class that uses for even records